go1.x/hello/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -8,7 +8,7 @@ Description: >
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
   Function:
-    Timeout: 5
+    Timeout: 10
```

Comment:
I know I had initially mentioned that this should be a relatively safe change, but should we consider duplicating this project on the SAM CLI side, and instead point `sam init` to that project using the `--location` flag?

Reply:
Yeah, we can do that. Better to minimize the impact